### DISCUSSION TITLE
Dynamically add new Bot in Runtime

### DISCUSSION THREAD

**Discussion by script696** [bukhalo/nestjs-telegraf#1150] [Q&A]
<img width="795" alt="Снимок экрана 2023-09-29 в 16 56 31" src="https://github.com/evilsprut/nestjs-telegraf/assets/99768179/cbc92dee-223a-4320-9736-e2d8084873a3">

Help me pls 
I run multiple bots in my app according to Doc and i work fine
But is it possible to add new bot in runtime ?

I want to add a new bot through the admin panel by recording its name and token in the database.
After the array of bots in the database has been updated, you need to somehow register it in the application...

**Comment by bukhalo** [comment]
Ref: https://github.com/robot-mafia/nestjs-telegraf/discussions/1212#discussioncomment-10122699
